src/includes/performance-monitoring/configuration/javascript.mdx

```diff
@@ -199,34 +210,37 @@ When Sentry captures transactions, they are assigned a transaction name. This na
 An example of doing this in a node.js application:
 
 ```javascript
-import { addGlobalEventProcessor } from '@sentry/node';
+import { addGlobalEventProcessor } from "@sentry/node";
 
 addGlobalEventProcess(event => {
   // if event is a transaction event
   if (event.type === "transaction") {
-	event.transaction = sanitizeTransactionName(event.transaction);
+    event.transaction = sanitizeTransactionName(event.transaction);
   }
   return event;
 });
 ```
 
-For browser JavaScript applications using the `Tracing` integration, the `beforeNavigate` option  can be used to better group navigation/pageload transactions together based on URL.
+For browser JavaScript applications using the `BrowserTracing` integration, the `beforeNavigate` option can be used to better group navigation/pageload transactions together based on URL.
```

Comment:
@HazAT - this is a use of `beforeNavigate` which isn't covered by `tracesSampler`. We may need to rethink the idea of deprecating the former in favor of the latter.